### PR TITLE
Standardized setup

### PR DESCRIPTION
**Pull request by matthuhiggins**
* Add a dummy app!
* Introduces logs, to see what the tests are doing!
* Remove manual setup of models and db!
* Allows running of index.rake tasks from the project root!
* Introduces new way to define the percolator index up front, so that it's not created on demand!

@ebarendt @alanpearson

### CODE REVIEW

File: lib/elastic_record/index.rb [hunk 1]
```diff
@@ -81,8 +82,10 @@ def get(end_path, json = nil)
     end
 
     private
+
       def new_index_name
         "#{alias_name}_#{(Time.now.to_f * 100).to_i}"
```
Comment (ebarendt) [inline lib/elastic_record/index.rb]: Can we use `yyyymmdd` instead of `to_f`?

File: lib/elastic_record/index/percolator.rb [hunk 2]
```diff
@@ -2,9 +2,9 @@ module ElasticRecord
   class Index
     module Percolator
       def create_percolator(name, elastic_query)
-        unless exists? percolator_name
-          create percolator_name
-        end
+        # unless exists? percolator_name
```
Comment (ebarendt) [inline lib/elastic_record/index/percolator.rb]: Need to create the percolator first?

File: test/helper.rb [hunk 3]
```diff
@@ -1,14 +1,8 @@
-require 'bundler/setup'
-Bundler.require
+ENV["RAILS_ENV"] = "test"
```
Comment (ebarendt) [inline test/helper.rb]: Isn't this automagic?

**Comment by matthuhiggins** [comment]
No reason to create the percolator like that anymore. It will be created with `rake index:create` now, as long as your model has:

```ruby
class Product
  elastic_index.has_percolator = true
end
```

File: lib/elastic_record/index.rb (hunk 1 above)
Comment (matthuhiggins) [inline lib/elastic_record/index.rb]: I would probably go with dashes, "yyyy-mm-dd"

**Pull request merged**